backend/src/main/java/com/springBoot/fruits_ecommerce/repositorys/RoleRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.springBoot.fruits_ecommerce.repositorys;
+
+import com.springBoot.fruits_ecommerce.enums.RoleName;
```

Comment:
I believe security branch has been merged here as well, generally branches should not overlap with each other, as it will cause source code conflicts